README.md

```diff
@@ -101,6 +101,7 @@ Versions of supported components
 -   [weave](http://weave.works/) v2.3.0
 -   [docker](https://www.docker.com/) v17.03 (see note)
 -   [rkt](https://coreos.com/rkt/docs/latest/) v1.21.0 (see Note 2)
+-   [cert-manager](https://github.com/jetstack/cert-manager/releases) v0.3.0
```

Comment:
those are components listed not addons

Reply:
Oops, so shall we also add another section for addons?